models/Node.js

```diff
@@ -1,14 +1,15 @@
 const Data = require('./Data.js');
+const availableModules = require('../libraries/availableModules');
 
 /**
  * Constructor used to define every nodes generated in the Object. It does not need to contain its own ID
  * since the object is created within the nodes with the ID as object name.
  *
  * @constructor
  */
-function Node() {
+function Node(name, type) {
     // the name of each link. It is used in the Reality Editor to show the IO name.
-    this.name = '';
+    this.name = name;
```

Comment:
Might be good for this to be `name || ''` in case something's doing a `name === ''` or equivalent